lib/rfc3164.js

```diff
@@ -3,6 +3,7 @@
 const moment = require('moment-timezone')
 const stringify = require('fast-safe-stringify')
 const through2 = require('through2')
+const EOL = require('os').EOL
```

Comment:
Should this be configurable, e.g. `{ newline: '\n' }`? Or can we safely assume that `\n` will always be an acceptable terminator? I think so since syslog is a Unix tool and `\n` is the standard EOL for Unix.